gallery/src/styles/Nivo/NivoPieExample.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { ResponsivePie } from '@nivo/pie';
+import { NIVO_COLORS } from '@sonatype/react-shared-components';
+import { pieData } from './data';
+
+export default function NivoPieExample() {
```

Comment:
The black text on the pie slices isn't very readable, we probably shouldn't demo it like that.

Reply:
> The black text on the pie slices isn't very readable, we probably shouldn't demo it like that.

I removed them since you can still see the value when you hover over a slice, but we can also change the color or move the text to be outside of the slices? d60f20884941654946289eea32fa0a71170d611a